bicycleparameters/main.py

```diff
@@ -5,10 +5,14 @@
 
 # dependencies
 import numpy as np
+from numpy import pi, sin, cos
```

Comment:
You should use `np.pi`, `np.sin`, `np.cos` instead of bringing these in separately.